src/test/ui/borrowck/borrowck-access-permissions.ast.nll.stderr

```diff
@@ -35,7 +35,7 @@ error[E0596]: cannot borrow `*ptr_x` as mutable, as it is behind a `*const` poin
   --> $DIR/borrowck-access-permissions.rs:56:23
    |
 LL |         let ptr_x : *const _ = &x;
-   |                                -- help: consider changing this to be a mutable pointer: `&mut x`
+   |             ----- help: consider changing this to be a mutable pointer: `*mut i32`
```

Comment:
this is interesting; hypothetically we probably would prefer, if possible to highlight the ascribed *type*`*const _` here, right?